app/security.py

```diff
@@ -46,17 +40,22 @@
         'https://www.gstatic.com',
         'https://cdn.ons.gov.uk'
     ],
+
 }
 
 
 DEFAULT_RESPONSE_HEADERS = {
     'Strict-Transport-Security': 'max-age=31536000 includeSubDomains',
     'Content-Security-Policy': CSP,
     'X-Content-Security-Policy': CSP,
-    'X-XSS-Protection': '1; mode=block',
     'X-Frame-Options': 'DENY',
     'X-Content-Type-Options': 'nosniff',
-    'Referrer-Policy': 'strict-origin-when-cross-origin',
+    'Referrer-Policy': 'no-referrer',
+    'X-Permitted-Cross-Domain-Policies': 'None',
+    'clear-site-data': ['cache', 'cookies', 'storage'],
+    'Cross-Origin-Opener-Policy': 'same-origin',
+    'Cross-Origin-Resource-Policy': 'same-site',
+    'Cache-Control': ['no-store', 'max-age=0']
```

Comment:
```suggestion
    'Cache-Control': ['no-store', 'max-age=0'],
    'Server': 'Office For National Statistics',
```

need to set the Server header to stop exposing internal tech info